test/langtools/tools/javac/options/HelpOutputColumnWidthTest.java

```diff
@@ -47,7 +47,7 @@
 
 public class HelpOutputColumnWidthTest extends TestRunner {
 
-    public static final int MAX_COLUMNS = 84;
+    public static final int MAX_COLUMNS = 80;
```

Comment:
I though that the default terminal width was 24x80 in most systems, so 84 seems an odd choice.  I agree that 80 seems more appropriate.

Reply:
oops.. that should have been `80x24`.